### PR TITLE
Feature/referee

### PR DESCRIPTION
**Pull request by PhiBabin**
- Add handling of the referee command
- Fix autoplay
- Add forwarding of autoplay to ui-debug
- Fix problem where geometry message was receive, but not used
- Remove exec() from stateless tactic
- Remove sim_yellow.cfg, because I can
- Fix dual_launch.sh
- Fix a bug where ReceiverProcesses would silently crash, instead of printing the exception and its backtrace.

### CODE REVIEW

File: ai/Algorithm/auto_play.py [hunk 1]
```diff
@@ -113,25 +114,24 @@ def _normal_start(self):
...
-    def _select_next_state(self):
-        referee = GameState().referee
+    def _select_next_state(self, ref_state: RefereeState):
         next_state = self.current_state
         # On command change
-        if self.last_ref_command != referee.command:
+        if self.last_ref_state != ref_state.command:
             next_state = {
-                RefereeCommand.HALT: SimpleAutoPlayState.HALT,
+                InternalRefereeCommand.HALT: SimpleAutoPlayState.HALT,
```
Comment (sim0504) [inline ai/Algorithm/auto_play.py]: whats the difference with internal command?

File: ai/executors/play_executor.py [hunk 2]
```diff
@@ -41,11 +46,13 @@ def exec(self) -> List[EngineCommand]:
...
         """
 
-        # if self.play_state.autonomous_flag:
-        #     if GameState().game.referee.team_info['ours']['goalie'] != self.goalie_id:
-        #         self.goalie_id = GameState().game.referee.team_info['ours']['goalie']
-        #         GameState().update_player_for_locked_role(self.goalie_id, Role.GOALKEEPER)
-        #     self.auto_play.update(self._has_available_players_changed())
+        self._fetch_referee_state()
+
+        # TODO: Add a warning when no ref has been received since the start
```
Comment (sim0504) [inline ai/executors/play_executor.py]: Make this an issue

File: ai/Algorithm/auto_play.py (hunk 1 above)
Comment (PhiBabin) [reply ai/Algorithm/auto_play.py]: RefereeCommand -> BLABLA_YELLOW, BLABLA_BLUE
InternalRefereeCommand -> BLABLA_US, BLABLA_THEM

File: ai/executors/play_executor.py (hunk 2 above)
Comment (PhiBabin) [reply ai/executors/play_executor.py]: Done

File: ai/Algorithm/auto_play.py (hunk 1 above)
Comment (PhiBabin) [reply ai/Algorithm/auto_play.py]: RefereeState s'occupe de la conversion, mais c'est vrai que l'on pourrait avoir un constreur de InternalRefereeCommand qui prend en entrée une RefereeCommand et qu'il le convertie.
Présentement, RefereCommand est uniquement utilisé par RefereeState.

**Pull request merged**